package.json

```diff
@@ -95,20 +95,22 @@
     "intl-list-format": "^1.0.3",
     "intl-messageformat": "^9.4.6",
     "io-ts": "^2.2.9",
-    "jest": "28.1.3",
+    "jest": "29.4.3",
+    "jest-environment-jsdom": "~28.1.1",
```

Comment:
Why do we need this `jest-environment-jsdom` package now? And why it isn't on v29 like rest of Jest packages?